tests/publiccloud/download_repos.pm

```diff
@@ -32,8 +32,8 @@ sub run {
     select_host_console();    # select console on the host, not the PC instance
 
     # Skip maintenance updates. This is useful for debug runs
-    # Note: QAM_PUBLICCLOUD_SKIP_DOWNLOAD is left for backwards compatability and will be removed in the future
-    my $skip_mu = get_var('PUBLIC_CLOUD_SKIP_MU', get_var('QAM_PUBLICCLOUD_SKIP_DOWNLOAD', 0));
+    # Note: PUBLIC_CLOUD_SKIP_DOWNLOAD is left for backwards compatability and will be removed in the future
```

Comment:
Instead of replacing `QAM_PUBLICCLOUD_SKIP_DOWNLOAD` you could directly remove it. This is not used anymore as the comment suggests

```suggestion
    my $skip_mu = get_var('PUBLIC_CLOUD_SKIP_MU', 0);
```